docs/source/notebooks/tutorial.ipynb

```diff
@@ -752,14 +892,14 @@
    "source": [
     "As you can see, the search results above include `source_ids` for which we only have one of the two variables, and one or two of the three experiments.\n",
     "\n",
-    "We can tell `intake-esm` to discard any `source_id` that doesn't have both variables `[\"thetao\", \"o2\"]` and all three experiments `[\"historical\", \"ssp245\", \"ssp585\"]` by passing `require_all_on=[\"source_id\"]` to the search method:\n",
+    "We can tell intake ESM to discard any `source_id` that doesn't have both variables `[\"thetao\", \"o2\"]` and all three experiments `[\"historical\", \"ssp245\", \"ssp585\"]` by passing `require_all_on=[\"source_id\"]` to the search method:\n",
```

Comment:
Is "intake ESM" intended here?   I thought maybe the original was more accurate?